rpa-builder/6.7/modules/ROOT/pages/toolbox-mail-operations-mail-session-oauth.adoc

```diff
@@ -0,0 +1,142 @@
+= Mail Session (OAuth)
+
+The _Mail Session (OAuth)_ Action Step is an inclusive element for email operations that enables you to establish a connection to an email server using MS Outlook or Gmail and then perform email operations such as *Read Mail*, *Send Mail*, and *Set Mail* (to change email) by using the corresponding Action Steps.
+
+== Before you Begin
+
+Before you configure this action step, ensure you have: 
+
+* An Azure account, if you are creating an application for Outlook.
+* A Google account, if you are creating an application for Gmail.
+
+[[properties]]
+== Properties
+
+include::partial$oauth-credentials-properties.adoc[tags=common-oauth-properties]
+
+* *Incoming Mail Operation* Settings
+** *Protocol used*
++
+(Mail Protocol) The communication protocol to use: IMAP, Exchange, or POP3.
+** *Encryption used*
++
+(Encryption) The encryption protocol to use: None, SSL/TLS, or STARTTLS.
+** *Server address*
++
+(String) The server address entered in the wizard.
+** *Server Port*
++
+(Integer) The selected port.
+
+* *Outgoing Mail Operation* Setting
+**  *Encryption used*
++
+(Encryption) The encryption protocol to use: None, SSL/TLS, or STARTTLS.
+** *SMTP Server address*
++
+(String) The URL of the selected SMTP email server.
+** *SMTP Server Port*
++
+(Integer) The selected port.
+** *Send test mail to*
++
+(String) The Email address to which the test email is to
+be sent.
+
+== Inbound Variables
+
+* *SMTP Server address*
+* *Server address*
+* *SMTP Server Port*
+* *Server Port*
+* *OAuth Credentials*
+* *Auth Endpoint* 
+* *Token Endpoint* 
+* *OAuth Client ID*
+* *Client Secret* 
+* *Redirect URI*
+* *Scopes* 
+* *E-Mail Address*
+* *Refresh Token*
+
+== Outbound Variables
+
+The _Mail Session_ Action Step has no outbound variables.
+
+== Wizard
+
+image:mail-session-outlook-oauth-wizard.png[The Mail Session (OAuth) Action Step Wizard, 60%, 60%]
```

Comment:
best practice: alt text in quotes. Also, I think we don't use resize parameters?

Reply:
Should Wizard be lowercase here?